.github/scripts/build_mac.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Usage:
+#
+#     $ sh .github/scripts/build_mac.sh
+#     $ open _build/app_prod/rel/Livebook.app
+#     $ open livebook://github.com/livebook-dev/livebook/blob/main/test/support/notebooks/basic.livemd
+#     $ open ./test/support/notebooks/basic.livemd
+set -e
+
+sh app_builder/scripts/bootstrap-mac.sh
```

Comment:
Ideally we'd use off the shelf Mac builds hosted by the OTP team (or Hex team, etc) but let's build them themselves in the meantime.

Reply:
Should we keep said script in the same .github folder then? 🤔 Or do we want to provide it long term?

Reply:
My thinking was it is generic and thus more appropriate for `app_builder` which we'll eventually extract out but yeah, since it is ideally short lived, Livebook's .github folder is perhaps a better place indeed. Will change it!